src/components/common/hooks/FileReader.ts

```diff
@@ -22,6 +23,12 @@ export const useFileReader = () => {
         if (!fileData || fileData.data == null || !validator) {
             return
         }
+
+        // If the MIME type is not there
+        // In case of windows, the mime type gives some unexpected results so it needs to be handle explicitly
+        if (!fileData.type) {
+            fileData.type = getFileMimeType(fileData)
```

Comment:
We should not and can not mutate state directly it set itself throught setState.